processor/src/main/java/org/mapstruct/ap/internal/util/Message.java

```diff
@@ -225,8 +229,10 @@ public enum Message {
     ANNOTATE_WITH_ENUM_CLASS_NOT_DEFINED( "enumClass needs to be defined when using enums." ),
     ANNOTATE_WITH_ENUMS_NOT_DEFINED( "enums needs to be defined when using enumClass." ),
     ANNOTATE_WITH_ANNOTATION_IS_NOT_REPEATABLE( "Annotation \"%s\" is not repeatable." ),
-    ANNOTATE_WITH_DUPLICATE( "Annotation \"%s\" is already present with the same elements configuration.", Diagnostic.Kind.WARNING ),
-    ;
+    ANNOTATE_WITH_DUPLICATE(
+        "Annotation \"%s\" is already present with the same elements configuration.",
+        Diagnostic.Kind.WARNING
+    );
```

Comment:
Please do not commit unrelated style changes. Can you please remove this change?